src/renderer/base/renderer.cpp

```diff
@@ -774,8 +785,35 @@ void Renderer::_PaintBufferOutputHelper(_In_ IRenderEngine* const pEngine,
             // If we're allowed to do grid drawing, draw that now too (since it will be coupled with the color data)
             if (_pData->IsGridLineDrawingAllowed())
             {
-                // We're only allowed to draw the grid lines under certain circumstances.
-                _PaintBufferOutputGridLineHelper(pEngine, currentRunColor, cols, screenPoint);
+                // If we found a wide character while we looped above, it's possible we skipped over the right half

```

Comment:
please link this back up to the github issue it closes -- that's important context for "why we decided we had to do this"